src/lib/widget/inline.svelte

```diff
@@ -20,71 +20,16 @@
 
 <script lang="ts">
   import { createEventDispatcher, onMount } from 'svelte';
-  import type { z } from 'zod';
 
   import Journey from '$journey/journey.svelte';
-
-  // Import the stores for initialization
-  import configure from '$lib/sdk.config';
-  import { initialize as initializeJourneys } from '$journey/config.store';
-  import { initialize as initializeJourney } from '$journey/journey.store';
-  import { initialize as initializeContent } from '$lib/locale.store';
-  import { initialize as initializeLinks, partialLinksSchema } from '$lib/links.store';
-  import { initialize as initializeOauth } from '$lib/oauth/oauth.store';
-  import { initialize as initializeUser } from '$lib/user/user.store';
-  import { initialize as initializeStyle } from '$lib/style.store';
-
-  import type { partialConfigSchema } from '$lib/sdk.config';
-  import type { journeyConfigSchema } from '$journey/config.store';
-  import type { partialStringsSchema } from '$lib/locale.store';
-  import type { partialStyleSchema } from '$lib/style.store';
-
-  export let config: z.infer<typeof partialConfigSchema> | undefined = undefined;
-  export let content: z.infer<typeof partialStringsSchema> | undefined = undefined;
-  export let journeys: z.infer<typeof journeyConfigSchema> | undefined = undefined;
-  export let links: z.infer<typeof partialLinksSchema> | undefined = undefined;
-  export let style: z.infer<typeof partialStyleSchema> | undefined = undefined;
```

Comment:
This is all the store related stuff pulled out of the Widget and moved into the API utility.